File: ru.kmorozov.library.data.client/src/main/java/ru/kmorozov/library/data/client/LibraryClientController.java
package ru.kmorozov.library.data.client;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

/**
 * Created by sbt-morozov-kv on 17.01.2017.
 */

@Controller
public class LibraryClientController {

    @RequestMapping("/")
    public String index() {
        return "index";
    }
}

File: ru.kmorozov.library.data.client/src/main/java/ru/kmorozov/library/data/client/LibraryClientController.kt
package ru.kmorozov.library.data.client

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * Created by sbt-morozov-kv on 17.01.2017.
 */

@Controller
class LibraryClientController {

    @RequestMapping("/")
    fun index(): String {
        return "index"
    }
}
